addons/im_livechat/static/src/embed/common/mail_popout_service_patch.js

```diff
@@ -0,0 +1,15 @@
+import { patch } from "@web/core/utils/patch";
+import { loadAssets } from "@im_livechat/embed/common/boot_helpers";
+import { mailPopoutService } from "@mail/core/common/mail_popout_service";
+
+const popoutPatch = {
+    async addAssets(window) {
+        await super.addAssets(...arguments);
+        // FIXME: without this, fonts detected on the document are incorrect,
+        // they seem to default to the font list of the parent window,
+        // even though this same promise is awaited before checking the fonts in `loadFont`
+        await window.document.fonts.ready;
+        await loadAssets(window.document.head, window.document);
+    },
+};
```

Comment:
The function to patch should receive `options` or `context` instead of `window`, so that it can have proper `if (env.embedLivechat)` branch to apply livechat assets specifically for livechat.

I'm ok with loading fonts in all cases on `window`, pretty sure that's needed all the time for Odoo `popout` feature.

Reply:
Matthieu suggested earlier to put this patch in live chat's `src/embed/` directory so that the branching based on `env.embedLivechat` is not necessary (since the assets / this patch) is only loaded when we are in embed